src/main/java/org/fogbowcloud/saps/engine/scheduler/restlet/DatabaseApplication.java

```diff
@@ -91,6 +91,7 @@ public Restlet createInboundRoot() {
 		router.attach("/regions/details", RegionResource.class);
 		router.attach("/regions/search", RegionResource.class);
 		router.attach("/email", ProcessedImagesResource.class);
+		router.attach("/processedTasks", ProcessedTasksResource.class);
```

Comment:
@velmer I would change it to finishedTasks or archivedTasks. Something related to the state machine we use in SAPS

Reply:
archivedTasks makes more sense, I'll change it